src/Public/Security.ps1

```diff
@@ -132,36 +129,24 @@ function Add-PodeSecurityHeader {
 
         [Parameter()]
         [switch]
-        $Add,
-
-        [Parameter()]
-        [ValidateSet('Upper', 'Lower')]
-        [string]
-        $Casing
+        $Append
     )
 
     if (![string]::IsNullOrWhiteSpace($Value)) {
-        if ($add -and $PodeContext.Server.Security.Headers.ContainsKey($Name)) {
+        if ($Append -and $PodeContext.Server.Security.Headers.ContainsKey($Name)) {
             $Headers = $PodeContext.Server.Security.Headers[$Name].split(',')
-            if ($Headers -contains $Value) {
-                return
-            } else {
+            if ($Headers -inotcontains $Value) {
                 $Headers += $Value
                 $PodeContext.Server.Security.Headers[$Name] = (($Headers.trim() | Select-Object -Unique) -join ', ')
+            } else {
+                $return
             }
```

Comment:
`return` instead of `$return` 👀 though tbh, I'd remove the else block, there's no logic after this that could run that needs to be skipped.

Reply:
done